### ISSUE TITLE
Add select all button on the bookmark page

### ISSUE THREAD

**Issue by onaralili** [onaralili/SplitUp#14]
Save some tabs then go to bookmark page using the bookmark button on top of the extension. We would like to add Select All button next to Trash button (please keep the style: the color of the button etc.). 
The expected behavior is when a user clicks Select All button, all the checkboxes must be selected.

<img width="406" alt="image" src="https://user-images.githubusercontent.com/5898068/46683627-f357b080-cbf0-11e8-932a-e12c94781450.png">

This feature has been requested in #12

**Comment by agampreetsingh** [comment]
Hi @onaralili !
Wanted to confirm if the select all button selects all the check boxes or the check boxes of the 'Split' that the select-all button is placed in...

**Comment by agampreetsingh** [comment]
Since the issue requires a 'Select-all' button with the 'Trash' button, the user would expect it to work for individual windows only. 
You require it work for all check boxes at once, shouldn't the 'Select-all' button be placed at the top independently (not beside the 'Trash' button)? If so, please tell the desired location for it.
I hope I have not understood this incorrectly. Enlighten me if it is the case...

**Comment by onaralili** [comment]
> Since the issue requires a 'Select-all' button with the 'Trash' button, the user would expect it to work for individual windows only.
> You require it work for all check boxes at once, shouldn't the 'Select-all' button be placed at the top independently (not beside the 'Trash' button)? If so, please tell the desired location for it.
> I hope I have not understood this incorrectly. Enlighten me if it is the case...

Agree. It makes sense. Originally, the feature request was to have a button to select all the checkboxes, however, it seems like it requires some UI changes. I think at this point as you pointed out, select all button will select all the checkboxes within the window.

**Comment by agampreetsingh** [comment]
> > Since the issue requires a 'Select-all' button with the 'Trash' button, the user would expect it to work for individual windows only.
> > You require it work for all check boxes at once, shouldn't the 'Select-all' button be placed at the top independently (not beside the 'Trash' button)? If so, please tell the desired location for it.
> > I hope I have not understood this incorrectly. Enlighten me if it is the case...
> 
> Agree. It makes sense. Originally, the feature request was to have a button to select all the checkboxes, however, it seems like it requires some UI changes. I think at this point as you pointed out, select all button will select all the checkboxes within the window.

There is no action for the same if we have 'Selected all checkboxes in a window' then. The only action available is the action on the 'Trash' button, which already does the same i.e it deletes an entire window, similar to what is expected to happen when we 'select-all' and delete.
I think the correct way would be to put the button independently on the top if it is really required.
Kindly correct me if I am wrong.
